cinema/helper.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pathlib
+import uuid
+from typing import Optional, TYPE_CHECKING
+from django.utils.text import slugify
+
+if TYPE_CHECKING:
+    from cinema.models import Movie
```

Comment:
by doing this you create circular dependency -> here you import Movie from models and in models you import movies_image_path from helper